cloud-init/write-kali-aws-config.tpl.sh

```diff
@@ -13,8 +13,8 @@
 # * aws_region - the AWS region where the roles are to be assumed
```

Comment:
Since this has been dragged in I just want to mention that I think the findings data AWS credentials should be modified to be like the artifact export credentials. Specifically the script name and the name of the generated AWS credentials file. This can be done in a separate PR via an issue. I just want to reach consensus on the suggestion.

Reply:
Yes, I am planning to do that after this PR is merged, but I hadn't mentioned it here yet.  I wasn't planning on changing the script name, but I was definitely going to rename the credentials file.  I intentionally want the names of these scripts to be somewhat different, in order to minimize confusion.